lib/main.dart

```diff
@@ -1,125 +1,145 @@
+      'https://image.tmdb.org/t/p/w500/qNBAXBIQlnOThrVvA6mA2B5ggV6.jpg',
+      'https://image.tmdb.org/t/p/w500/9n2tJBplPbgR2ca05hS5CKXwP2c.jpg',
+      '2023-04-05',
+      '7,8',
+      'While working underground to fix a water main, Brooklyn plumbers—and brothers—Mario and Luigi '
+          'are transported down a mysterious pipe and wander into a magical new world. But when the '
+          'brothers are separated, Mario embarks on an epic quest to find Luigi.');
+
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'Flutter Demo',
-      theme: ThemeData(
-        // This is the theme of your application.
-        //
-        // TRY THIS: Try running your application with "flutter run". You'll see
-        // the application has a blue toolbar. Then, without quitting the app,
-        // try changing the seedColor in the colorScheme below to Colors.green
-        // and then invoke "hot reload" (save your changes or press the "hot
-        // reload" button in a Flutter-supported IDE, or press "r" if you used
-        // the command line to start the app).
-        //
-        // Notice that the counter didn't reset back to zero; the application
-        // state is not lost during the reload. To reset the state, use hot
-        // restart instead.
-        //
-        // This works for code too, not just values: Most code changes can be
-        // tested with just a hot reload.
-        colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
-        useMaterial3: true,
-      ),
-      home: const MyHomePage(title: 'Flutter Demo Home Page'),
-    );
+        theme: ThemeData(
+          colorScheme: ColorScheme.fromSwatch().copyWith(
+            primary: Colors.black12,
+          ),
+        ),
+        home: Scaffold(
+            backgroundColor: Colors.cyan,
+            appBar: AppBar(
+              title: Text(
+                m.title,
+              ),
+              centerTitle: true,
+            ),
+            body: SingleChildScrollView(
+              child: SafeArea(
+                  top: true,
+                  bottom: true,
+                  left: true,
+                  right: true,
+                  child: Center(
+                    child: Column(
+                      children: [
+                        Stack(
+                          children: [
+                            Image.network(m.backdrop),
+                            Positioned(
+                              bottom: 0,
+                              right: 20,
+                              child: Text(
+                                m.originalTitle,
+                                style: const TextStyle(
+                                  fontWeight: FontWeight.bold,
+                                  fontSize: 25,
+                                  color: Colors.white,
+                                ),
+                                textAlign: TextAlign.center,
```

Comment:
You could extract this as a standalone widget

Reply:
done